scripts/build-cache-key.js

```diff
@@ -0,0 +1,89 @@
+"use strict";
+const path = require("path");
+
+const HELPER_MODULE = "@babel/helper-caching";
+const CORE_MODULE = "@babel/core";
+
+module.exports = {
+  swapPackageWithKey,
+  buildCacheKey,
+};
+
+function swapPackageWithKey(srcPath) {
+  return srcPath.replace(/package.json$/, path.join("lib", "_cache-key.js"));
+}
+
+function buildCacheKey(pkgPath, pkgContent) {
+  const json = JSON.parse(pkgContent);
+  const name = json.name;
+  const version = json.version;
+  const scripts = json.scripts;
+  const peerDependencies = json.peerDependencies || {};
+  const dependencies = json.dependencies || {};
+
+  const deps = Object.keys(dependencies).reduce((acc, name) => {
+    if (name === "babylon" || /^@babel\//.test(name)) {
+      acc.push(`require(${JSON.stringify(name)}).CACHE_KEY`);
+    }
+
+    return acc;
+  }, []);
+
+  let content;
+
+  if (name === HELPER_MODULE) {
```

Comment:
Shouldn't this be `if (name === HELPER_MODULE || name === "babylon") {`? (Based on the comment below)

Reply:
Ah woops, I need to edit the comment, good catch. I had Babylon in here and then removed it because I realized this doesn't work with Babylon because it uses Rollup.